src/data.json

```diff
@@ -2,31 +2,71 @@
   "cards": [
     {
       "name": "francrepas",
-      "src": "/images/cards/francrepas.jpeg"
+      "src": "/images/cards/francrepas.webp"
     },
     {
       "name": "harbinger",
-      "src": "/images/cards/harbinger.jpeg"
+      "src": "/images/cards/harbinger.webp"
     },
     {
       "name": "holofernes",
-      "src": "/images/cards/holofernes.jpeg"
+      "src": "/images/cards/holofernes.webp"
     },
     {
       "name": "mouillevent",
-      "src": "/images/cards/mouillevent.jpeg"
+      "src": "/images/cards/mouillevent.webp"
     },
     {
       "name": "painensac",
-      "src": "/images/cards/painensac.jpeg"
+      "src": "/images/cards/painensac.webp"
     },
     {
       "name": "papeligosse",
-      "src": "/images/cards/papeligosse.jpeg"
+      "src": "/images/cards/papeligosse.webp"
+    },
+    {
+      "name": "epistemon",
+      "src": "/images/cards/epistemon.webp"
+    },
+    {
+      "name": "esquire",
+      "src": "/images/cards/esquire.webp"
+    },
+    {
+      "name": "eudemon",
+      "src": "/images/cards/eudemon.webp"
+    },
+    {
+      "name": "grangousier",
+      "src": "/images/cards/grangousier.webp"
+    },
+    {
+      "name": "jobelin",
+      "src": "/images/cards/jobelin.webp"
+    },
+    {
+      "name": "wetgullet",
+      "src": "/images/cards/wetgullet.webp"
+    },
+    {
+      "name": "eabbe",
+      "src": "/images/cards/eabbe.webp"
+    },
+    {
+      "name": "flutter",
+      "src": "/images/cards/flutter.webp"
+    },
+    {
+      "name": "gargamelle",
+      "src": "/images/cards/gargamelle.webp"
+    },
+    {
+      "name": "jobeliness",
+      "src": "/images/cards/jobeliness.webp"
     }
   ],
   "cover": {
     "name": "cover",
-    "src": "/images/cards/cover.png"
+    "src": "/images/cards/cover.webp"
```

Comment:
`webp` formatı kalite olarak iyi görünürken boyut olarak çok daha optimize bir imaj formatı. Fakat kullandığımız şeylerin tarayıcı desteklerini bilmemiz iyi olur.

`caniuse` sitesinde her şeyin tarayıcı uyumluluğuna bakabilirsin. Örneğin IE11 desteğini önemsemesek bile webp imajların orada doğru çalışmayacağını biliyoruz bu sayede.
https://caniuse.com/?search=webp